packages/ui/src/primitives/FormatCurrency.tsx

```diff
@@ -17,11 +17,16 @@ const FormatCurrency: FC<ComponentPropsWithoutRef<typeof Text> & Props> = ({
 
   useEffect(() => {
     if (amount) {
+      const lowestValue = Number(
```

Comment:
don't really get what's going on here

Reply:
Basically finding the lowest value possible according to the maximumFractionDigits and then checking if the amount is lower than the maximum so we can display `< $0.01` instead of `$0.00`

Reply:
Ah cool!